src/llamafactory/chat/hf_engine.py

```diff
@@ -86,22 +92,37 @@ def _process_args(
             and image is not None
             and not hasattr(processor, "image_seq_length")
             and template.image_token not in messages[0]["content"]
-        ):  # llava-like models
+            and model_args.visual_inputs_type in ["vision_tower","phi3v_like"]
+        ):
+            # llava-like models
             messages[0]["content"] = template.image_token + messages[0]["content"]
+        elif image is not None and model_args.visual_inputs_type == "qwenvl_like":
+            # Add image pathlike token as vision input
+            image_path = pathlib.Path(DEFAULT_CACHE_DIR) / "temp.png"
```

Comment:
If two or more people are visiting this API, how can such code cope with that situation? You can change it to timestamp or find another way

Reply:
I will use BytesIO to fix this.I think this will work and save image filelike obj in memory is more easy to cleanup.How do you think?

Reply:
![image](https://github.com/hiyouga/LLaMA-Factory/assets/45327989/de8edf87-9b1a-4c22-9f67-d4ff35d6077c)
Yep it will work.

Reply:
Ah it cant work due to a force change in modeling_qwen.py

Reply:
Okay, you can use uuid